app/docs/0.13.x/proxy.md

```diff
@@ -604,150 +648,148 @@ Will be proxied upstream by Kong as:
 Host: example.com
 ```
 
-And this request will match api-2:
+And this request will match the second one:
 
 ```http
 POST / HTTP/1.1
 Host: example.com
 ```
 
-Following this logic, if a third API was to be configured with a `hosts` field,
-a `methods` field, and a `uris` field, it would be evaluated first by Kong.
+Following this logic, if a third Route was to be configured with a `hosts`
+field, a `methods` field, and a `uris` field, it would be evaluated first by
+Kong.
 
 [Back to TOC](#table-of-contents)
 
 ## Proxying behavior
 
 The proxying rules above detail how Kong forwards incoming requests to your
-upstream services. Below we detail what happens internally between the time
-Kong *recognizes* an HTTP request to a target service, and the actual
-*forwarding* of the request upstream.
+upstream services. Below, we detail what happens internally between the time
+Kong *matches* an HTTP request with a registered Route, and the actual
+*forwarding* of the request.
 
 [Back to TOC](#table-of-contents)
 
 ### 1. Load balancing
 
-Starting with Kong 0.10, Kong implements load balancing capabilities to
-distribute the forwarded requests across multiple instances of an upstream
-service.
+Kong implements load balancing capabilities to distribute proxied
+requests across a pool of instances of an upstream service.
 
-Previous to Kong 0.10, Ordinarily, Kong would send proxied requests to the
-`upstream_url`, and load balancing across multiple upstream instances required
-an external load balancer.
-
-You can find more informations about adding load balancing to your APIs by
-consulting the [Load Balancing Reference][load-balancing-reference].
+You can find more information about configuring load balancing by consulting
+the [Load Balancing Reference][load-balancing-reference].
 
 [Back to TOC](#table-of-contents)
 
 ### 2. Plugins execution
 
-Kong is extensible via "plugins" that hook themselves in the
-request/response lifecycle of the proxied requests. Plugins can perform a
-variety of operations in your environment and/or transformations on the proxied
-request.
+Kong is extensible via "plugins" that hook themselves in the request/response
+lifecycle of the proxied requests. Plugins can perform a variety of operations
+in your environment and/or transformations on the proxied request.
+
+Plugins can be configured to run globally (for all proxied traffic) or on
+specific Routes and Services. In both cases, you must create a [plugin
+configuration][plugin-configuration-object] via the Admin API.
 
-Plugins can be configured to run globally (for all proxied traffic) or on a
-per-API basis by creating a [plugin configuration][plugin-configuration-object]
-through the Admin API.
+Once a Route has been matched (and its associated Service entity), Kong will
+run plugins associated to either of those entities. Plugins configured on a
+Route run before plugins configured on a Service, but otherwise, the usual
+rules of plugins association apply.
```

Comment:
suggestion: is it possible to add a link here to where  these rules of plugin association are explained?